src/index.tsx

```diff
@@ -60,15 +61,14 @@ export function useSpeechRecognition(options: SpeechRecognitionOptions = default
       if (recognition) {
         switch (disconnectType) {
           case SpeechRecognitionDisconnectType.RESET:
-            dispatch(setStatus(SpeechRecognitionStatus.RESET));
-            dispatch(setPauseAfterDisconnect(false));
             recognition.abort();
+            dispatch(disconnectAction(SpeechRecognitionStatus.RESET));
```

Comment:
the order of the abort call and the dispatch has changed.  Intentional?

Reply:
When I consolidated everything I felt it made more sense to wait until after it was off to change the status. It's ultimately inconsequential though I believe.